### DISCUSSION TITLE
Getting Branches to same level as master

### DISCUSSION THREAD

**Discussion by frius94** [community/community#22662] [New to GitHub]
<p>Hi I am learning to use GIT and have a problem and I’m not sure how to solve it.</p>
<p>I have my feature and development branch behind the master and want them to be on the same level as the master branch. Which command can I use to do that? git rebase wouldn’t solve my problem, right?</p>
<p>Thank you for your help.</p>
<p><img src="https://user-images.githubusercontent.com/38991482/181072530-e885827d-6800-4275-88b2-5c81f15b8825.png" alt="git.PNG" data-base62-sha1="fIWkFfDm8FWpv7rrMs8TPizqyrA" width="563" height="266"></p>

**Comment by Wabri** [comment, marked as answer]
<p>Hi <a class="mention" href="/frius94">@frius94</a>,</p>
<p>You can do with the git reset command. Checkout to the branch you want to put at the same level of master, for example “development”:</p>
<pre><code>git checkout development
</code></pre>
<p>Than reset with hard mode:</p>
<pre><code>git reset --hard master
</code></pre>
<p>That’s all.</p>
<p>Hope this helps :slightly_smiling_face:  </p>
<p>-Gabriele-</p>
<blockquote>
<p><strong>Mark helpful posts with Accept as Solution to help other users locate important info. Don’t forget to give Kudos for great content!</strong></p>
</blockquote>

**Comment by mimi011** [comment]
<p>For reset and i am a master</p>

**Comment by enowmbi** [comment]
<p>Thanks for the info.</p>
